Library/Formula/flex.rb

```diff
@@ -1,20 +1,26 @@
 class Flex < Formula
   homepage "http://flex.sourceforge.net"
-  url "https://downloads.sourceforge.net/flex/flex-2.5.37.tar.bz2"
-  sha1 "db4b140f2aff34c6197cab919828cc4146aae218"
+  url "https://downloads.sourceforge.net/flex/flex-2.5.39.tar.bz2"
+  sha1 "267794c709d5c50f2dcb48ff5d8dcbbfe40f953e"
 
   bottle do
     revision 1
-    sha1 "0a2bb0ce9a49330e5fd40b6e409a353972cf8840" => :yosemite
-    sha1 "0dd7fc9c36a6258b2e456e7dd0c5818d07e2a2ea" => :mavericks
-    sha1 "69b5f449a9c0bf5fd37f999dca5ccfd120a6f389" => :mountain_lion
+    sha1 "5f0433bfc16766d83789d741d968f057617d5c96" => :yosemite
+    sha1 "f56a2926b77c560de0b4eb75a8cd56191711339e" => :mavericks
+    sha1 "df8067013c8dfd099b42164cf58e05dccfed7ea3" => :mountain_lion
   end
 
   keg_only :provided_by_osx, "Some formulae require a newer version of flex."
 
   depends_on "gettext"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool"  => :build
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and the reason why it is needed as a comment in the formula file. Thanks!